javatools/build.gradle.kts

```diff
@@ -75,25 +76,39 @@ val jar by tasks.existing(Jar::class) {
     inputs.property("manifestVersion") {
         version
     }
-    archiveBaseName = "javatools"
 
-    // TODO: It may be fewer special cases if we just add to the source sets from these dependencies, but it's not
-    //  apparent how to get that correct for includedBuilds.
-    from(xdkJavaToolsUtils)
+    /*
+     * This "from" statement will copy everything in the dependencies section, and our resources
+     * to the javatools-<version>.jar. In that respect it's a fat jar.
+     *
+     * The "copyDependencies" task will output a build directory containing all our dependencies,
+     * which are currently just javatools-utils-<version>.jar and jline.
+     *
+     * Map semantics guarantee that we resolve the "from" input only during the execution phase.
+     * We take the destination directory, known to be an output of copyDependencies, and a
+     * single directory. This is implicit from the "into" configuration of that task.
+     * Then we lazily (with "map") assume that every file in the destination tree is a jar/zip file
+     * (we will break if it isn't) and unpack that into the javatools jar that is being built.
+     *
+     * TODO: an alternative solution would be to leave the run-time dependent libraries "as is" and
+     *      use the "Class-Path" attribute of the manifest to point to them.
+     */
+    from(copyDependencies.map { fileTree(it.destinationDir).map { jarFile -> zipTree(jarFile) }})
+
+    archiveBaseName = "javatools"
 
     manifest {
         attributes(
             "Manifest-Version" to "1.0",
-            "Xdk-Version" to semanticVersion.toString(),
-            "Sealed" to "true",
             "Main-Class" to "org.xvm.tool.Launcher",
             "Name" to "/org/xvm/",
             "Specification-Title" to "xvm",
             "Specification-Version" to version,
             "Specification-Vendor" to "xtclang.org",
             "Implementation-Title" to "xvm-prototype",
             "Implementation-Version" to version,
-            "Implementation-Vendor" to "xtclang.org"
+            "Implementation-Vendor" to "xtclang.org",
+            "Sealed" to "true"
         )
```

Comment:
Where did the Xdk-Version go? You have just made it impossible for the plugin to determine which javatools it may have on the runtime classpath.

Reply:
I didn't understand the reason of that to be. Why was it called Xdk-Version? This is a javatools.jar, what does it have to do with the XDK?

Reply:
I'll revert, but it needs to be documented